pkg/ccip/bindings/common/common.go

```diff
@@ -164,11 +184,22 @@ func (c *CrossChainAddress) LoadFromCell(s *cell.Slice) error {
 
 // LoadCrossChainAddressWithoutPrefix parses a CrossChainAddress from raw data if lacks a length prefix as the first byte.
 func LoadCrossChainAddressWithoutPrefix(s *cell.Slice) (CrossChainAddress, error) {
-	data, err := s.LoadSlice(s.BitsLeft())
+	bitsLeft := s.BitsLeft()
```

Comment:
Ignoring the auditor suggestion for checking the byte-alignment here, more [info](https://smartcontract-it.atlassian.net/browse/NONEVM-3310?focusedCommentId=515119)